script/build_docker.sh

```diff
@@ -16,9 +16,12 @@ echo "$TRAVIS_COMMIT" > REVISION
 
 docker build -t quay.io/$TRAVIS_REPO_SLUG:$TRAVIS_COMMIT .
 
-docker run --net host quay.io/$TRAVIS_REPO_SLUG:$TRAVIS_COMMIT rake db:create db:migrate
-docker run -d --net host quay.io/$TRAVIS_REPO_SLUG:$TRAVIS_COMMIT
-sleep 10
+docker run -e RAILS_ENV=production -e SECRET_KEY_BASE=1234 --net host quay.io/$TRAVIS_REPO_SLUG:$TRAVIS_COMMIT rake db:create db:migrate
```

Comment:
:question:

Reply:
> uninitialized constant RuboCop (NameError) 

Should have been caught with ping test we do in this script. It wasn't because by default it was using the development environment. 
This change ensures that we use production env (unless rake.